source/PrintData.cpp

```diff
@@ -648,6 +665,75 @@ namespace {
 			if(filter.Matches(&it.second))
 				cout << it.first << '\n';
 	}
+
+
+	void Phrases(const char *const *argv)
+	{
+		string phraseName;
+
+		for(const char *const *it = argv + 1; *it; ++it)
+		{
+			string arg = *it;
+			if(arg == "--phrase" && *++it)
+			{
+				phraseName = *it;
+				break;
+			}
+		}
+
+		const Phrase *phrase = nullptr;
+
+		for(const auto &it : GameData::Phrases())
+			if(it.first == phraseName)
+			{
+				phrase = &it.second;
+				break;
+			}
+
+		if(!phrase)
+		{
+			cout << "Error: invalid phrase name: \"" + phraseName + "\".\n";
+			return;
+		}
+
+		cout << "Computing results for phrase: \"" << phrase->Name() << "\"." << endl;
```

Comment:
These messages seem to be unnecessary, and would only be annoying for anybody that wants to parse the output. So I say let's remove them.

Reply:
Done